source/base/Text.ooc

```diff
@@ -134,8 +134,8 @@ Text: cover {
 	free: func@ ~withCriteria (criteria: Owner) -> Bool {
 		this _buffer free(criteria)
 	}
-	slice: func (start, distance: Int) -> This {
-		result := This new(this _buffer slice(start, distance))
+	slice: func (start: Int, distance := INT_MAX) -> This {
+		result := This new(this _buffer slice(start, distance == INT_MAX ? this count - start : distance))
```

Comment:
Nothing here. The change @simonmika talks about above would have to be implemented in the buffers.